### PR TITLE
Add : Add small text mode #44

### PR DESCRIPTION
**Pull request by lucas-jg**

### CODE REVIEW

File: src/context/MainContext.jsx [hunk 1]
```diff
@@ -15,6 +15,15 @@ export const MainProvider = ({ children }) => {
     JSON.parse(localStorage.getItem("darkTheme")) || false
   );
 
+  const [isSmallText, setIsSmallText] = useState(
+    JSON.parse(localStorage.getItem("smallText")) || false
```
Comment (max-programming) [inline src/context/MainContext.jsx:19]: Nice! But I think we should keep it enabled on mobile view. You can use `useMediaQuery` hook for that and change the default value

**Review by max-programming — CHANGES_REQUESTED** [review]

File: src/context/MainContext.jsx (hunk 1 above)
Comment (lucas-jg) [reply src/context/MainContext.jsx:19]: I'll request PR again by tomorrow.

Comment (lucas-jg) [reply src/context/MainContext.jsx:19]: `useMedaiQuery` doesn't work in `MainContext`. So I solved it in `Todo` component.

**Comment by max-programming** [comment]
Awesome @lucas-jg. But it conflicts because of some of my recent commits. Please resolve those so that I can merge

**Comment by max-programming** [comment]
Encountered a small issue that on mobile, the small text looks fine. 
image

But when we goto Settings, the setting exists but off. Either remove the setting on mobile view or make it enabled
image

**Review by max-programming — APPROVED** [review]

**Pull request merged**